source/auth/auth.rst

```diff
@@ -678,7 +678,10 @@ authMechanism
 	Sets the Mechanism property on the MongoCredential. When not set, the default will be one of SCRAM-SHA-256, SCRAM-SHA-1 or MONGODB-CR, following the auth spec default mechanism rules.
 
 authSource
-	Sets the Source property on the MongoCredential. This overrides the database name on the connection string for where authentication occurs. The default is admin.
+	Sets the Source property on the MongoCredential.
```

Comment:
Is this the only clarifying point needed in the spec? In the spec ticket discussion it seemed like more was required.

Reply:
I believe this is - its just about what the default values should be and whether or not the database name should be used.  Individual mechanisms later declare the validity of the sources.